### ISSUE TITLE
Rename repository to be small letters only

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/model_seg_gm-wm_t2star_7T_unet3d-multiclass#1]
According to our [convention](https://github.com/ivadomed/ivadomed/wiki/repositories), all repos should be with small letters, so i suggest renaming as `model_seg_gm-wm_t2star_7T_unet3d-multiclass`

**Comment by naga-karthik** [comment]
Minor correction - the title of the repo should be `model_seg_gm-wm_t2star_7t_unet2d-multiclass` based on the description in the ReadMe.

**Comment by jcohenadad** [comment]
good point, it’s 2d not 3d. I’ve checked the JSON files. @Nilser3 can you please fix that?

**Comment by jcohenadad** [comment]
and we should probably remove the mention of unet and kernel dimensions all together in case futur models end up being 3d.
